x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rules_list.test.tsx

```diff
@@ -333,8 +333,11 @@ beforeEach(() => {
   (getIsExperimentalFeatureEnabled as jest.Mock<any, any>).mockImplementation(() => false);
 });
 
+// This entire test suite is flaky/timing out and has been skipped.
```

Comment:
cc @mikecote @XavierM

Reply:
Note that having the tests in this file enabled triggers an error deep in react-dom for the jest environment, which prevents the entire triggers_actions_ui test suite from executing. See https://github.com/elastic/kibana/issues/141477 for some historical notes on this error, and Constance left a comment at the end with her results from digging through these tests.